events/wc_titanforged_events.txt

```diff
@@ -1 +1,218 @@
+	}
+
+	desc = {
+		trigger = {
+			character = event_target:target_thorim
+		}
+		text = EVTDESC_WCUTF.1_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+	
+
+	trigger = {
+		OR = {
+			character = 400501
+			character = 400502
+		}
+		is_save_game = no
+	}
+
+	immediate = {
+		с_400501 = {
+			save_event_target_as = target_loken
+		}
+		c_400502 = {
+			save_event_target_as = target_thorim
+		}
+	}
+	option = {
+		name = EVTOPTA_WCUTF.1
+		trigger = { ai = yes }
+	}
+	option = {
+		name = EVTOPTB_WCUTF.1
+		trigger = {
+			OR = {
+				character = event_target:target_loken
+				AND = {
+					character = event_target:target_thorim
+					event_target:target_loken = { ai = yes }
+				}
+			}
+		}
+		character_event = { id = WCUTF.2 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+	option = {
+		name = EVTOPTC_WCUTF.1
+		trigger = {
+			OR = {
+				character = event_target:target_loken
+				AND = {
+					character = event_target:target_thorim
+					event_target:target_loken = { ai = yes }
+				}
+			}
+		}
+		character_event = { id = WCUTF.3 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.2
+	desc = {
+		trigger = {
+			character = event_target:target_loken
+		}
+		text = EVTDESC_WCUTF.2_loken
+	}
+
+	desc = {
+		trigger = {
+			character = event_target:target_thorim
+		}
+		text = EVTDESC_WCUTF.2_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.2
+		trigger = {
+			character = event_target:target_loken
+		}
+		religion = old_gods_worship
+	}
+	option = {
+		name = EVTOPTB_WCUTF.2
+		trigger = {
+			character = event_target:target_thorim
+		}
+		religion = old_gods_worship
+		event_target:target_loken = {
+			this_abdicate_transfer_wealth_to_prev_effect = yes
+			
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		d_ulduar = { any_de_jure_vassal_title = { location = { remove_province_modifier = ulduar_closed_down } } }
+		set_global_flag = ulduar_opened_flag
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.3
+	desc = {
+		trigger = {
+			character = event_target:target_loken
+		}
+		text = EVTDESC_WCUTF.3_loken
+	}
+
+	desc = {
+		trigger = {
+			character = event_target:target_thorim
+		}
+		text = EVTDESC_WCUTF.3_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.3
+		trigger = {
+			character = event_target:target_loken
+		}
+		event_target:target_thorim = {
+			event_target:target_loken = {
+				this_abdicate_transfer_wealth_to_play_prev_effect = yes
```

Comment:
Thorim died, but the event tries to switch. Thus, I get game over though I had Mimiron as my heir.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78928194-3c701580-7ab1-11ea-89ce-f2b0810e7344.png)
![image](https://user-images.githubusercontent.com/46384992/78928255-57db2080-7ab1-11ea-9242-d7de0bc81342.png)

</details>

Reply:
how did he die?

Reply:
Via the console, not the event's fault.

Reply:
yeah but how would he die in real game?

Reply:
> yeah but how would he die in real game?

It doesn't matter. The thing is the event isn't dying-proof.

Reply:
you can only break it with console tho

Reply:
> you can only break it with console tho

@sejtalk challenge accepted.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78930172-b2c24700-7ab4-11ea-9ef9-1e936d6f1641.png)
![image](https://user-images.githubusercontent.com/46384992/78930190-be157280-7ab4-11ea-936d-205a3cc1916c.png)

</details>